linux_pipeline/Jenkinsfile_upstream_kernel

```diff
@@ -423,8 +438,6 @@ pipeline {
                                        string(credentialsId: 'AZURE_SAS', variable: 'AZURE_SAS'),
                                        string(credentialsId: 'UPSTREAM_AZURE_STORAGE_URL', variable: 'UPSTREAM_AZURE_STORAGE_URL'),
                                        string(credentialsId: 'LISA_TEST_DEPENDENCIES', variable: 'LISA_TEST_DEPENDENCIES'),
-                                       string(credentialsId: 'DB_CONFIG_UPSTREAM', variable: 'DB_CONFIG_UPSTREAM'),
```

Comment:
leave this as they are, they are not the purpose of this commit.
if you prefer to change, you need to create a special commit/PR for the changes